Jenkinsfile

```diff
@@ -111,7 +114,11 @@ pipeline {
 									cd -
 
 									npm run serve -- $WORKSPACE/app/build -p 300$N -a 127.0.0.1 &>server.log &
-									npm run cypress:run -- --record
+									set +e
+									npm run cypress:run -- --record |tee cypress.log
+									ret=$?
```

Comment:
`$?` is the return code from `tee ...` not from `npm run cypress:run ...`, so `exit 0` even if `2 of 13 failed`  in https://jenkins.lisk.io/job/lisk-hub/job/PR-1389/3/console

Not sure if there is a way to get the return code. If not, we can grep `cypress.log` for `All specs passed!`

Reply:
@slaweet you're right, let's set `pipefail`